pkgs/applications/networking/onionshare/default.nix

```diff
@@ -124,6 +124,7 @@ rec {
         inherit tor meek obfs4 snowflake;
         inherit (tor) geoip;
       })
+      ./fix-qrcode-gui.patch
```

Comment:
shall we use `fetchpatch2` instead of vendoring the patch?

Reply:
I have no strong feelings on this issue tbh.